servers.json

```diff
@@ -4,7 +4,8 @@
     "url": "https://usegalaxy.eu",
     "libs": {
       "GTN.yaml": "7d84c2ac21245dad",
-      "EI.yaml": "cef78de78c2ec365"
+	"EI.yaml": "cef78de78c2ec365",
+	"SILVA.yaml": "F50c98c8a05e8bc97"
```

Comment:
```suggestion
      "EI.yaml": "cef78de78c2ec365",
      "SILVA.yaml": "F50c98c8a05e8bc97"
```

Reply:
The SILVA ID has been modified because it was wrong. The correct one is 40160950115d555e.

Reply:
my suggestion is w/r/t the spacing, not the ID.

Reply:
Yes, thanks!